src/evaluate.c

```diff
@@ -0,0 +1,79 @@
+#include "jou_compiler.h"
+
+
+const struct SpecialVar specialVars[] = {
```

Comment:
this probably bad idea, simpler to call `evaluate_condition()` whenever this would be needed